app/sequencescape_excel/sequencescape_excel/specialised_field/dual_index_tag_well.rb

```diff
@@ -0,0 +1,59 @@
+# frozen_string_literal: true
+
+module SequencescapeExcel
+  module SpecialisedField
+    ##
+    # DualIndexTagWell
+    class DualIndexTagWell
+      # 2 tags in the well, 1 tag from tag_group and 1 tag from tag2_group
+      TAGS_PER_WELL = 2
+
+      include Base
+      include ValueRequired
+      include ValueToUpcase
```

Comment:
Might be worth adding a comment against this `ValueToUpcase` to explain that `well_index` would return a different value for 'A1' vs 'a1'.